docs/ops/activation/PReLU_1.md

```diff
@@ -2,32 +2,59 @@
 
 **Versioned name**: *PReLU-1*
 
-**Category**: Activation function
+**Category**: *Activation function*
 
-**Short description**: *PReLU* performs element-wise parametric ReLU operation with negative slope defined by the second input.
+**Short description**: Parametric rectified linear unit element-wise activation function.
 
-**Attributes**: operation has no attributes.
+**Detailed description**
 
-**Inputs**
+*PReLU* operation is introduced in this [article](https://arxiv.org/pdf/1502.01852v1.pdf).
 
-* **1**: `X` - Input tensor of any supported floating point type T1. Required.
+*PReLU* performs element-wise parametric *ReLU* operation on a given input tensor, based on the following mathematical formula:
 
-* **2**: `slope` - Tensor with negative slope values of type T2. The shape of the tensor should be broadcastable to input 1. Required.
+\f[
+PReLU(x) = \left\{\begin{array}{r}
+    x \quad \mbox{if } x \geq  0 \\
+    \alpha x \quad \mbox{if } x < 0
+\end{array}\right.
+\f]
 
-**Outputs**
+Where α, is a learnable parameter and corresponds to the negative slope defined by the second input `slope`.
 
-* **1**: The result of element-wise PReLU operation applied for tensor from input 1 with slope values from input 2. A tensor of type T1 and shape matching shape of input *x* tensor.
+Before the operation computation, the second input tensor `slope` is broadcasted to the first input tensor `data` based on [Broadcast Rules For Elementwise Operations](../broadcast_rules.md). Additionally, *PReLU* is equivalent to *ReLU* operation when `slope` is equal to zero input tensor.
```

Comment:
It is not true. The PReLU `slope` input is applied per-channel. So, the input tensor with shape `[1,32,46,46]` the slope shape will be `[32]`. So, we had a bug in the PReLU operation which you can fix now as part of the operation revise.

Reply:
I have addressed the comment as requested. Please re-review